catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/SecurityAttributes.java

```diff
@@ -74,6 +74,56 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 BasicTypes.STRING_TYPE));
+
+        /* Experimental Attributes, subject to change*/
+        DESCRIPTORS.add(new AttributeDescriptorImpl(METADATA_ORIGINATOR_CLASSIFICATION,
```

Comment:
@codice/security - I wanted to make sure everyone was aware of the "ext" attributes used here to store metadata and resource security markings for MGMP.  Also, be aware these will also be used by the NSILI transformer.